Lib/datetime.py

```diff
@@ -1025,15 +1025,82 @@ def __repr__(self):
 
 
     def ctime(self):
-        "Return ctime() style string."
+        """Return ctime() style string.
+
+        >>> date(2001, 10, 5).ctime()
+        'Fri Oct  5 00:00:00 2001'
+        """
         weekday = self.toordinal() % 7 or 7
         return "%s %s %2d 00:00:00 %04d" % (
             _DAYNAMES[weekday],
             _MONTHNAMES[self._month],
             self._day, self._year)
 
     def strftime(self, fmt):
-        "Format using strftime()."
+        """Format date or datetime given the format string `fmt`
```

Comment:
I think these changes are overkill for docstrings, and they impose a maintenance burden to keep them up to date. How about just containing a reference to the actual documentation?

Reply:
Perhaps a good compromise could be including the most common ones, and then having a link to the remaining ones?

Reply:
@ericvsmith bump?

Reply:
Are there any examples of functions (preferably in datetime) that contain pointers to their documentation? Or is it just assumed that people know how to look it up?

I'm not crazy about trying to decide what common codes would be.

You should probably reach out to the module maintainer for more input on this. I'm just an interested bystander.  @pganssle @abalkin: Any thoughts?

Reply:
Having the full documentation be available inline would be the optimal solution, but I'd settle for a quick link - as long as there's something to help avoid the frustrating web search and scroll through the massive amount of online documentation. It could be interesting to look into generating documentation from docstrings and the like (using a tool like pdoc) so that there would be as much difficulty in terms of keeping documentation up-to-date in multiple places, but I'd say that's well beyond the scope of this particular PR.